tests/pypeline/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+add_library(fakePipeboxImpl SHARED FakePipeboxImpl.cpp)
+add_library(fakePipebox SHARED FakePipebox.cpp)
```

Comment:
1. Use `revng_add_library`.
2. Do not deviate from our library naming conventions.

Reply:
1. I'd really like to not have it installed in `root`